tools/sst_dump_test.cc

```diff
@@ -116,8 +117,14 @@ class SSTDumpToolTest : public testing::Test {
 
     // Populate slightly more than 1K keys
     uint32_t num_keys = kNumKey;
-    for (uint32_t i = 0; i < num_keys; i++) {
-      tb->Add(MakeKey(i), MakeValue(i));
+    if (reverse) {
+      for (uint32_t i = num_keys; i > 0; i--) {
```

Comment:
In this case, 0 is not tested

Reply:
I avoided  it because the condition i>=0 will cause the infinite loop since i is unsigned.
I will change the i type int32_t